### PR TITLE
[AssetMapper] Exclude dot files

### LINKED ISSUE #19182: [AssetMapper] Exclude dot files
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#52712
| PR author(s) | @weaverryan
| Merged in    | 6.4

We created this issue to not forget to document this new feature. We would really appreciate if you can help us with this task. If you are not sure how to do it, please ask us and we will help you.

To fix this issue, please create a PR against the 6.4 branch in the [symfony-docs repository](https://github.com/symfony/symfony-docs).

Thank you! :smiley:

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Fix #19182

### CODE REVIEW

File: frontend/asset_mapper.rst [hunk 1]
```diff
@@ -936,6 +936,25 @@ This is a list of glob patterns that will be excluded from the asset map:
...
+
+.. code-block:: yaml
+
+    framework:
+        asset_mapper:
+            exclude_dotfiles: true
+
+This option is enabled by default.
+
+.. versionadded:: 7.1
```
Comment (smnandre) [inline frontend/asset_mapper.rst:971]: 6.4 (just in time 😅 )

**Comment by OskarStark** [comment]
Thank you Alexandre.

File: frontend/asset_mapper.rst (hunk 1 above)
Comment (OskarStark) [reply frontend/asset_mapper.rst:971]: Fixed while merging

Comment (alexandre-daubois) [reply frontend/asset_mapper.rst:971]: Oh good catch, thanks!

**Pull request merged**